glean/metrics.yaml

```diff
@@ -332,6 +332,55 @@ sample:
           `authenticationinapp.h`.
         type: string
 
+  auth_account_deletion_step:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      The user has advanced in the authentication flow for the account deletion.
+    bugs:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/issues/2956
+    data_reviews:
+      - TBD
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - amarchesini@mozilla.com
+    expires: never
+    extra_keys:
+      state:
+        description: |
+          The name of the authentication state, e.g. "sign-in", "sign-out",
+          "totp", etc. The list of states can be found in
+          `authenticationinapp.h`.
+        type: string
+
+  auth_subscription_mgmt_step:
```

Comment:
I'd like to mirror `authentication_inapp_step`. My suggestion is `authentication_sub_manage_step`.

Reply:
Updated the name.